### PR TITLE
add autoFocus to input when filter, fix time filter 300 -> 500

### PR DESCRIPTION
**Pull request by sanglevinh**

### CODE REVIEW

File: frontend/src/components/Database/DatabaseCells.tsx [hunk 1]
```diff
@@ -31,21 +31,27 @@ type CellProps = {
...
+              sx={{paddingTop: "16px"}}
+              defaultValue={item.value || ''}
+              onChange={(e) => {
                 if(timeout) clearTimeout(timeout)
                 timeout = setTimeout(() => {
                   applyValue({...item, value: e.target.value})
-            }, 300)
-          }
-          } />
+                }, 500)
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseCells.tsx:51]: @sanglevinh 
This is a minor point, but could you please keep this (waiting time) as a constant?

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

File: frontend/src/components/Database/DatabaseExperiments.tsx [hunk 2]
```diff
@@ -341,7 +343,7 @@ const DatabaseExperiments = ({
       items: [
         {
           field: Object.keys(dataParamsFilter).find(key => (dataParamsFilter as any)[key])?.replace('publish_status', 'published') || '' ,
-          operator: Object.keys(dataParamsFilter).find(key => (dataParamsFilter as any)[key]) === 'publish_status' ? 'is' : 'contains',
+          operator: ['publish_status', 'brain_area'].includes(Object.keys(dataParamsFilter).find(key => (dataParamsFilter as any)[key]) || 'publish_status') ? 'is' : 'contains',
```
Comment (itutu-tienday) [inline frontend/src/components/Database/DatabaseExperiments.tsx:346]: @sanglevinh 
The "No_published" in publish_status does not seem to be working. Please reconfirm.

**Review by itutu-tienday — CHANGES_REQUESTED** [review]

**Comment by itutu-tienday** [comment]
@sanglevinh 
Just a reminder, this PR has over 2,000 commits (due to rebase?). Is there any problem with this?

File: frontend/src/components/Database/DatabaseExperiments.tsx (hunk 2 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseExperiments.tsx:346]: @itutu-tienday I have reported to BE and it has been fixed, please check again

File: frontend/src/components/Database/DatabaseCells.tsx (hunk 1 above)
Comment (sanglevinh) [reply frontend/src/components/Database/DatabaseCells.tsx]: @itutu-tienday i have added it

**Comment by sanglevinh** [comment]
> @sanglevinh Just a reminder, this PR has over 2,000 commits (due to rebase?). Is there any problem with this?

@itutu-tienday Because I am using rebase, there is no issue with it.

**Review by itutu-tienday — APPROVED** [review]

**Pull request merged**